app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author_id, class_name: 'User', foreign_key: :author_id_id
+  has_many :comments
+  has_many :likes
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_comments_counter
+    update(comments_counter: comments.count)
+  end
+
+  def update_likes_counter
+    update(likes_counter: likes.count)
+  end
```

Comment:
- [ ] Kindly take these functions in their respective models (comments and likes model). A counter should be recorded in the comments and likes models respectively.

__HINT:__
```Ruby
def update_comments_counter
  post.increment!(:comments_counter)
end

def update_likes_counter
  post.increment!(:likes_counter)
end
```


![model](https://github.com/toyman640/blog/assets/53124548/55e22b1a-9074-4d8c-833a-802db565bfd1)